libs/stats/odc/stats/utils.py

```diff
@@ -292,3 +292,19 @@ def fuse_ds(
     fused_ds = Dataset(product, prep_eo3(fused_doc), uris=[""])
     fused_doc["properties"]["fused"] = "True"
     return fused_ds
+
+
+def install_ows_environment():
```

Comment:
I don't like this Sai.
- We want these as optional dependencies, and if we try to use a feature that absolutely depends on them we should fail with an error.
- I don't ever want tools to silently try to install extra libraries when I run them
- In many environments it would fail anyway.

Please remove this function and it's usage.

Reply:
As we (@omad and Sai) discussed, I will:

1. put the OWS relative dependency as optional dependencies in setup.cfg to manage.
2. make sure the import OWS will display useful message to let user know they should run pip install odc-stats[ows] (just example CMD here) to setup environment.

Reply:
Already did the change at https://github.com/opendatacube/odc-tools/pull/392/commits/5593cf223a6aa4382fd3835e22cb3a5e3e89dc9c